packages/lerna/schemas/lerna-schema.json

```diff
@@ -1703,6 +1706,10 @@
           "type": "boolean",
           "description": "During `lerna version`, when true, commit and tag version changes."
         },
+        "runScriptsOnLockfileUpdate": {
+          "type": "boolean",
+          "description": "During `lerna version`, when true, updates the lock file running the lifecycle scripts after the version bump."
```

Comment:
Can you please clarify this description a bit? "updates the lock file running the lifecycle scripts" is unclear. Maybe something like `... when true, runs lifecycle scripts when syncing the lock file after the version bump.`

Reply:
Done!